runelite-api/src/main/interfaces/interfaces.toml

```
@@ -129,6 +129,24 @@ coffer=2
 id=598
 container=2
 
+[castlewars_lobby]
+id=131
+time_remaining=2
+
+[castlewars_saradomin_game]
```

The "game" suffix is redundant, particularly since "game_timer" is one of the children.
```suggestion
[castlewars_saradomin]
```

---

Fixed